satsense/image.py

```diff
@@ -276,32 +272,48 @@ def save(self, filename_prefix='', extension='nc'):
                 self._save_as_netcdf(data, filename, window)
             elif extension.lower() == 'tif':
                 self._save_as_tif(data, filename, window)
+        return filenames
 
     def _save_as_netcdf(self, data, filename, window):
         """Save feature vector as NetCDF file."""
+        if len(data.shape) == 2:
+            height, width = data.shape
```

Comment:
This is not how I intended this case: it was meant for cases where you do not extract features on a rectangular grid, but at a list of arbitrary points in the image, e.g. if you want to extract the same number points from all classes to get a balanced training dataset.

Reply:
Perhaps that is a use case we need a different save/load mechanism for anyways.

Current implementation is 'grid-based' we can also add a 'sample-based' storage?

Reply:
Ok